code/modules/mob/living/living.dm

```diff
@@ -515,6 +515,10 @@
 	if(loc && (istype(loc, /obj/structure/morgue) || istype(loc, /obj/structure/crematorium)))
 		resist_tray(L)
 
+	// The doctor left the odysseus unlocked again
+	if(src.loc && (istype(loc, /obj/item/mecha_parts/mecha_equipment/tool/sleeper)))
+		resist_mecha()
```

Comment:
Going to second what Tiger said---we have a lot of snowflaked resist types, that would ideally be unified into resist_container()